### ISSUE TITLE
Users cant login to OpenEMR on pippin

### ISSUE THREAD

**Issue by leebc** [aethelwulffe/PQRS_Gateway#1]
Admin can login to OpenEMR on pippin, but there is a problem with other users.

**Comment by leebc** [comment]
There are 2 columns in the OpenEMR 4.2 users database that are NOT in the OpenEMR 4.3 users database.
physician_type VARCHAR(50) Default=NUll  UTF8_bin 
cpo3 TINYINT(1) Default=NUll
* Added these columns  to users database on pippin
* Set an access control level for all our users.
--> This should resolve the problem.  Meg is able to login, testuser is able to login.

**Comment by tmccormi** [comment]
Physician type was added in 4.3  did you run the upgrade script?

4_2_0-to-4_3_0_upgrade.sql:ALTER TABLE users ADD COLUMN physician_type
VARCHAR(50) DEFAULT NULL;

cpo3 must be custom. It has never existed in any instance I can find.


Tony McCormick, CTO
www.mi-squared.com
Support: 866-735-0897, Direct: 713-574-6709
My Calendar:  http://bit.ly/XznvDo
“I am and always will be the optimist. The hoper of far-flung hopes and the
dreamer of improbable dreams.” — The Doctor, Season 6



On Wed, Jan 27, 2016 at 3:37 PM, leebc <notifications@github.com> wrote:

> There are 2 columns in the OpenEMR 4.2 users database that are NOT in the
> OpenEMR 4.3 users database.
> physician_type VARCHAR(50) Default=NUll UTF8_bin
> cpo3 TINYINT(1) Default=NUll
>
>    - Added these columns to users database on pippin
>    - Set an access control level for all our users. --> This should
>    resolve the problem. Meg is able to login, testuser is able to login.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aethelwulffe/PQRS_Gateway/issues/1#issuecomment-175911479>
> .
>

-- 
Please be aware that e-mail communication can be intercepted in 
transmission or misdirected. Please consider communicating any sensitive 
information by telephone.  The information contained in this message may be 
privileged and confidential. If you are NOT the intended recipient, please 
notify the sender immediately with a copy to hipaa-security@mrsb-ltd.com and 
destroy this message.

**Comment by leebc** [comment]
Tony, thanks for the quick reply and the code snipit. :-)

At the moment we have 2 servers with OpenEMR 4.2.x installed  one with OpenEMR 4.3 installed.  These were fresh installs.  We had created users on the 4.2 server and I wanted to copy them to the 4.3 server.  So I made a backup of these tables: group, users, users_facility, users_secure, user_settings.
The SQl command included DROP commands so when I imported them to the 4.3 server, they were missing the physician_type column.
I think we are moving everything to 4.3, so this shouldn't be a problem again.
(Until it is.)
